docs/howto/db.md

```diff
@@ -10,10 +10,11 @@ through `database/sql`.
 To wrap a `sql.DB` with SQLair, use the `sqlair.NewDB` method. This will return
 a `sqlair.DB`.
 
-> See more:
+```{seealso}
 [`sqlair.NewDB`](https://pkg.go.dev/github.com/canonical/sqlair#NewDB),
```

Comment:
I suggest you stick to the manual method as I believe the directive only comes in one flavor -- "See also" -- while in practice it can be useful to have a 4-way distinction:

- See also
- See more
- See first
- See next

Reply:
I've got a way to do custom text there, you can do:
```
```{admonition} See more
:class: tip
```
And replace `See more` with whatever you like.

I've done this in this PR.